.github/merge_rules.json

```diff
@@ -163,14 +156,14 @@
       ]
    },
    {
-      "name": "MPS",
+      "name": "Apple M1/MPS",
       "patterns": [
          "test/test_mps.py",
          "aten/src/ATen/native/native_functions.yaml",
          "aten/src/ATen/mps/**",
          "aten/src/ATen/native/mps/**"
       ],
-      "approved_by": ["kulinseth", "razarmehr"],
+      "approved_by": ["kulinseth", "alband", "malfet"],
```

Comment:
Why do you want to remove @razarmehr from this list?

Reply:
added him back in the latest version